tests/FSharp.Compiler.ComponentTests/EmittedIL/GenericComparison/Compare07.fsx.il.net472.bsl

```diff
@@ -0,0 +1,765 @@
+
```

Comment:
I can see a lot of .bsl added with this PR.

E.g. for Compare07, we have:

il.bsl
ill.netcore.bsl
il.net472.bsl

What is the first one (without specified platform) supposed to indicate?

Reply:
that is a bug, really good eyes.  the issue is the ill.netcore should be il.netcore.bsl, and il.bsl shouldn't e xist.

Reply:
fixed